apps/hash-ai-worker-ts/src/activities/infer-entities.ts

```diff
@@ -357,7 +350,7 @@ const requestEntityInference = async (params: {
               {
                 role: "user",
                 content:
-                  "Your previous response contained invalid JSON. Please try again.",
+                  "Your previous response contained invalid  Please try again.",
```

Comment:
@CiaranMn I think this removal may have been unintentional (or should have been changed to something else?)

Reply:
It was, thank you. I'll fix